src/workerd/util/sqlite.c++

```diff
@@ -603,6 +607,13 @@ bool SqliteDatabase::isAuthorized(int actionCode,
             auto val = KJ_UNWRAP_OR(param2, return false);
             return regulator.isAllowedName(val);
           }
+          case PragmaSignature::NULL_NUMBER_OR_OBJECT_NAME: {
+            // Argument is not required
+            auto val = KJ_UNWRAP_OR(param2, return true);
+            // val might be "N", where N is an integer, which will pass the following
+            // check but isn't actually an object name.
```

Comment:
@kentonv is there a quick easy way to check a `kj::StringPtr` against a regexp?

Reply:
How about: `val.tryParseAs<uint>() != nullptr`

That's going to be a whole lot faster than setting up a regex engine.